content/vendor/snapshots/backend-stores.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2019-01-23
+linktitle: "Backend Stores"
+title: Backend Stores
+---
```

Comment:
This seems odd as a separate page. Maybe combine this with the vendor page for configuring snapshots? It also seems like there is an admin console component here.